pkg/knuu/knuu_old.go

```diff
@@ -0,0 +1,122 @@
+// Package knuu provides the core functionality of knuu.
+package knuu
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/celestiaorg/knuu/pkg/builder"
+	"github.com/celestiaorg/knuu/pkg/builder/docker"
+	"github.com/celestiaorg/knuu/pkg/builder/kaniko"
+)
+
+// This is a temporary variable to hold the knuu instance until we refactor knuu pkg
+// TODO: remove this temporary variable
+var tmpKnuu *Knuu
```

Comment:
Isn't this PR doing what this comment is talking about? 
Why is it still needed?